setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import find_packages
 import sys
 
-version = '5.0.16.dev0'
+version = '5.0.16'
```

Comment:
Please remove this version number change. PRs should not normally update the version number. This package is under development, so the 'dev' marker should stay.

Reply:
Good to know! Thank you very much for the feedback.